server/db/dbManager.ts

```diff
@@ -45,7 +53,22 @@ export class PostManager {
 		if (pageSize * (pageNum - 1) > count) {
 			return [];
 		}
-		return this.model.find().skip(pageSize * (pageNum - 1)).limit(pageSize).exec();
+		let sorted;
+		switch (sort) {
```

Comment:
You should always include default section for `switch`. When your enum will expand you will avoid an error if you miss it here.